content/components/autocomplete.mdx

```diff
@@ -6,83 +6,143 @@ status: Alpha
 import {LabelGroup, Label, Box} from '@primer/components'
 
 <Box sx={{fontSize: 3}} class="lead" as="p">
-    Autocomplete inputs allow users to quickly filter through a list of options to pick one or more values for a field.
+  Autocomplete allows users to quickly filter through a list of options and pick one or more values for a field.
 </Box>
 
-<LabelGroup display="block" mb={4}>
-    <Label as="a" href="https://primer.style/components/Autocomplete" variant="xl" color="text.success" outline style="text-decoration: none; line-height: 20px;">
-        <img width="20" height="20" alt=" " src="https://user-images.githubusercontent.com/293280/123878374-ce9d4d00-d8f3-11eb-8adf-1a160292ff53.png" style="vertical-align: middle; margin-right: 4px;" />
-        React
-    </Label>
-    {/* TODO: uncomment once Figma docs are live
-    <Label as="a" href="https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=1927%3A0" variant="xl" outline style="text-decoration: none; line-height: 20px;">
-        <img width="20" height="20" alt=" " src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png" style="vertical-align: middle; margin-right: 4px;" />
-        Figma
-    </Label> */}
+<LabelGroup display="flex" mb={4}>
+  <Label
+    as="a"
+    href="https://primer.style/components/Autocomplete"
+    variant="xl"
+    outline
+    display="flex"
+    style="text-decoration: none;  align-items: center;"
+  >
+    <img
+      width="20"
+      height="20"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878374-ce9d4d00-d8f3-11eb-8adf-1a160292ff53.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    React
+  </Label>
+  <Label
+    as="a"
+    href="https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=1927%3A0"
+    variant="xl"
+    outline
+    display="flex"
+    style="text-decoration: none; align-items: center;"
+  >
+    <img
+      width="20"
+      height="20"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Figma
+  </Label>
+  <Label
+    as="a"
+    href="https://primer.style/view-components/components/beta/autocomplete"
+    variant="xl"
+    outline
+    display="flex"
+    style="text-decoration: none; min-width: 10ch; justify-content: center;"
+  >
+    Rails
```

Comment:
This looks a little misaligned. Was it supposed to include a ViewComponent icon?

> ![Screenshot](https://user-images.githubusercontent.com/293280/171466305-5e1ad272-992d-404e-b856-02a10ad9891e.png)